pyproject.toml

```diff
@@ -1,55 +1,51 @@
 [build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
 
-[tool.poetry]
-name = "ch-tools"
+[project]
+name = "chtools"
```

Comment:
Is there a reason to eliminate the hyphen in the package name?

Reply:
Yes, flit doesn't accept project name otherwise. I guess we could change src/chtools to src/ch-tools as an option

Reply:
The reason is that flit's package name must be equal to directory name in src/